data/actionsmap/yunohost.yml

```diff
@@ -1477,6 +1477,39 @@ tools:
 
 
 #############################
+#         Migrations        #
+#############################
+migrations:
+    category_help: Manage migrations
+    actions:
+
+        ### migrations_list()
+        list:
+            action_help: List migrations
+            api: GET /migrations
+
+        ### migrations_migrate()
+        migrate:
+            action_help: Perform migrations
+            api: POST /migrations/migrate
+            arguments:
+              -t:
+                  help: target migration number (or 0), latest one by default
+                  type: int
+                  full: --target
+              -f:
+                  help: fake run the migration, use it only if you know what you are doing
```

Comment:
Havent looked yet at what it does, but `--skip` would be more intuitive for me too.